openapi/platform/platform.yaml

```
@@ -314,6 +314,12 @@ paths:
           in: query
           name: 'fields[businessLocations]'
           description: 'Filter businessLocations by the following fields: name, phoneNumbers, customerIdentifier, geoCoordinate, serviceAreaBusiness, hours, and updatedAt. These fields will be included in the response.'
+        - schema:
+            type: array
```

```suggestion
            type: array
            item:
                type:string
            
```
you can make arrayp[string] like this